pkgs/development/libraries/libjxl/default.nix

```diff
@@ -21,28 +21,19 @@
 
 stdenv.mkDerivation rec {
   pname = "libjxl";
-  version = "0.7.0";
+  version = "0.8.1";
 
   outputs = [ "out" "dev" ];
 
   src = fetchFromGitHub {
     owner = "libjxl";
     repo = "libjxl";
     rev = "v${version}";
-    sha256 = "sha256-9DBLQ/gMyy2ZUm7PCWYJ7XOzkgQM0cAewJZzMNo8UPs=";
+    sha256 = "sha256-WWuvUTMrlR6ePbEs01ulLnuMiUqGrh4qELWFh0QMaGU=";
     # There are various submodules in `third_party/`.
     fetchSubmodules = true;
   };
 
-  patches = [
-    # present in master
-    (fetchpatch {
```

Comment:
if you want to, you can remove `fetchpatch` from line 2, because it's now unused.

Reply:
Fixed in [6e579a856509bb30af20e78bcf5a6fe0712729d](https://github.com/NixOS/nixpkgs/commit/66e579a856509bb30af20e78bcf5a6fe0712729d)